docs/ocis/release_notes.md

```diff
@@ -0,0 +1,282 @@
+### Architecture components
+
+ownCloud Infinite Scale is built as a modular framework in which components can be scaled individually. It consists of
+
+- a user management service
+- a settings service
+- a storage backend service
+- Built-in IdP
+- Frontend
+- Application gateway/proxy
+
+These components can be deployed in a multi-tier deployment architecture. See the [documentation](https://owncloud.github.io/ocis/) for an overview of the services.
+
+### Operation modes
+
+#### Full Stack Server mode (with oCIS storage driver)
+@TODO
+
+#### Single services scaleouts
+@TODO
+
+#### Bridge mode with ownCloud 10 backend
+
+For the product transition phase, ownCloud Infinite Scale comes with an operation mode ("bridge mode") that allows a hybrid deployment to be created between both server generations to operate the new web frontend with ownCloud 10 and Infinite Scale in parallel. This setup allows the ownCloud Web frontend to operate with both server generations and provides the foundation to migrate users gradually to the new backend.
+
+**Requirements for the bridge mode**
+- ownCloud Server >= 10.6
+- [Open ID Connect](https://marketplace.owncloud.com/apps/openidconnect) is used for user authentication
+- The [Graph API](https://marketplace.owncloud.com/apps/graphapi) app is installed on ownCloud Server
+- The latest client versions are rolled-out to users (required for OpenID Connect support). See the [ownCloud Documentation](https://doc.owncloud.com/server/admin_manual/configuration/user/oidc/#owncloud-desktop-and-mobile-clients) for more information.
+
+See the [Documentation](<https://owncloud.github.io/ocis/deployment/owncloud10_with_oc_web/>) on how to deploy Infinite Scale in bridge mode.
+
+{{< hint "warning" >}}
+**Technology Preview**
+
+ownCloud Infinite Scale is currently in Technology Preview. The bridge mode should only be used in non-production environments.
+{{< /hint >}}
+
+### What to expect?
+
+This is the first promoted public release of ownCloud Infinite Scale, released as "Technical Preview". Infinite Scale is not yet ready for production installations. Technical audiences will be able to get a good understanding of the potential of ownCloud's new platform.
+
+Version 1.0.0 comes with the base functionality for sync and share with a much higher performance-, stability- and security-level compared to all available platforms. Based on ten years of experience in enterprise sync and share and a long standing collaboration with the biggest global science organizations this new platform will exceed what content collaboration is today.
+
+### How to get started?
+
+One of the most important objectives for oCIS was to ease the setup of a working instance dramatically. Since oCIS is built on Google's powerful GO language it supports the single-file-deployment: Installing oCIS 1.0.0 is as easy as downloading a single file, applying execution permission to it and get started. No more fiddling around with complicated LAMP stacks.
```

Comment:
```suggestion
One of the most important objectives for oCIS was to ease the setup of a working instance dramatically. Since oCIS is built on Google's powerful Go language it supports the single-file-deployment: Installing oCIS 1.0.0 is as easy as downloading a single file, applying execution permission to it and get started. No more fiddling around with complicated LAMP stacks.
```